php/geosearch.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+//This file is autogenerated. See modules.json and autogenerator.py for details
+
+/*
+    geosearch.php
+
+    MediaWiki API Demos
+    Demo of `Geosearch` module: Search for wiki pages nearby
+
+    MIT License
+*/
+
+$endPoint = "https://en.wikipedia.org/w/api.php";
+$params = [
+    "action" => "query",
+    "list" => "geosearch",
+    "gscoord" => "37.7891838|-122.4033522",
+    "gsradius" => "10000",
+    "gslimit" => "10",
+    "format" => "json"
+];
+
+$url = $endPoint . "?" . http_build_query( $params );
+
+$ch = curl_init( $url );
+curl_setopt( $ch, CURLOPT_RETURNTRANSFER, true );
+$output = curl_exec( $ch );
+curl_close( $ch );
+
+$result = json_decode( $output, true );
+var_dump( $result );
```

Comment:
Maybe place title here like in: https://github.com/wikimedia/mediawiki-api-demos/blob/5d6745ecd1cb61dcbbdb5b96547329e589dff390/python/geosearch.py#L35